skimage/measure/_regionprops.py

```diff
@@ -166,6 +167,23 @@ def extent(self):
         return self.moments[0, 0] / (rows * cols)
 
     @property
+    def feret_diameter(self):
+        from ..morphology.convex_hull import convex_hull_image
+        from ._find_contours import find_contours
+        from scipy.spatial.distance import pdist
```

Comment:
@stevendbrown I looked into this and the circular import should be fixed as soon as this PR is merged: https://github.com/scikit-image/scikit-image/pull/1787

You can then simply rebase your branch here based on the changes there. Thanks!